stable_baselines3/her/her_replay_buffer.py

```diff
@@ -211,7 +211,13 @@ def _sample_transitions(
         # Select which episodes to use
         if online_sampling:
             assert batch_size is not None, "No batch_size specified for online sampling of HER transitions"
-            episode_indices = np.random.randint(0, self.n_episodes_stored, batch_size)
+            # Do not sample the episode with index `self.pos` as the episode is invalid
```

Comment:
so if I get the issue right, we cannot sample this episode because it is not finished yet?

Reply:
Yes, exactly.